v1-preparations/REMAINING_GA_OBJECTS.MD

```diff
@@ -9,34 +9,34 @@ Status is one of:
+| APPLICATION PACKAGE         |   ❌    | -                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| APPLICATION ROLE            |   ❌    | [#3134](https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/3134) -                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| CONNECTION                  |   🚀   | -                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Nit: there are some trailing spaces. Do we have a ticket for introducing a markdown formatter/linter?

Reply:
These trailing spaces allows jetbrains ides to display the md tables correctly, they were added on purpose